packages/libs/react-ui/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,80 @@
+import { Tab } from './Tab';
+import { selectorLine, tabsContainer } from './Tabs.css';
+import { TabsPanels } from './TabsPanels';
+
+import React, { FC, ReactNode, useEffect, useRef, useState } from 'react';
+
+export interface ITabsProps {
+  children?: ReactNode;
+  defaultSelected?: string;
+}
+
+interface ITabsComposition extends FC<ITabsProps> {
+  Tab: typeof Tab;
+  Panels: typeof TabsPanels;
+}
+
+// eslint-disable-next-line react/prop-types
+export const Tabs: ITabsComposition = ({ children, defaultSelected = '' }) => {
+  const [selectedTab, setSelectedTab] = useState<string>(defaultSelected);
+  const containerRef = useRef<HTMLDivElement | null>(null);
+  const lineRef = useRef<HTMLSpanElement | null>(null);
```

Comment:
I think a more descriptive name could improve this. Maybe selectedUnderline or something